docs/reference/project/stack.mdx

```diff
@@ -0,0 +1,72 @@
+A nitric stack is described by its definition in a `nitric.yaml` file. A stack represents a collection of resources that operate with each other and are deployed together.
+
+## Properties
+
+| **name**    | **description**                                              | **type**                           |
+| ----------- | ------------------------------------------------------------ | ---------------------------------- |
+| name        | A descriptive name of the stack                              | `string`                           |
+| functions   | zero or more functions to be deployed                        | map of [function](./functions)     |
+| containers  | zero or more containers to be deployed                       | map of [container](./containers)   |
+| topics      | zero of more topics to be deployed                           | map of [topic](./topics)           |
+| buckets     | zero or more buckets to be deployed                          | map of [bucket](./buckets)         |
+| schedules   | zero or more schedules to trigger functions or containers on | map of [schedule](./schedules)     |
+| sites       | zero or more static sites to deploy                          | map of [site](./sites)             |
+| apis        | zero or more APIs to deploy                                  | map of [api](./api-gateways)       |
+| entrypoints | zero or more entrypoints to deploy                           | map of [entrypoint](./entrypoints) |
+
+## Full Example
+
+```yaml
+name: example-image-processor
+
+functions:
+	get-processed-image:
+		handler: functions/get.ts
+  store-image:
```

Comment:
Viewing this file in Github show red blocks:
https://github.com/nitrictech/docs/blob/9525e6f7045bc0c5b8f7c1d5bf87e88ad5676edd/docs/reference/project/stack.mdx
Not sure if this is intentional or a display issue on Github.

Reply:
Replaces tabs with spaces